### PR TITLE
Align prefix for stepped lists

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

Align stepped lists prefixes

[/docs/examples/patterns/lists/lists-stepped-detailed](https://vanilla-framework-4735.demos.haus/docs/examples/patterns/lists/lists-stepped-detailed)

## Screenshots

image

### CODE REVIEW

**Comment by bartaz** [comment]
As discussed with Lyubo:

- font size of number same as heading, on same baseline, same font style
- circle size of line height (or .5rem bigger)
- spacing between circle and heading fixed (1.5rem?)

**Comment by lyubomir-popov** [comment]
@bartaz mockups [here](https://www.figma.com/file/H6MSsN3taoXXOJCPUbIImQ/Design-System-Library?node-id=5744%3A12484&t=blA6N4yLsHWIDOhI-1), to confirm circles == line-height + .5rem

**Comment by ClementChaumel** [comment]
Updated: 
0 0 0 0_8101_docs_examples_patterns_lists_lists-stepped-detailed

**Comment by lyubomir-popov** [comment]
I wonder whether we even need the circle. Thjis looks good to me (increased margin-right to 1.5rem):

image

**Comment by bartaz** [comment]
> I wonder whether we even need the circle. Thjis looks good to me (increased margin-right to 1.5rem):

@lyubomir-popov No separator at all? Dot at least for consistency with default ordered list styling?

1. something
2. something

**Comment by bartaz** [comment]
@lyubomir-popov And don't forget about default "stacked" version, also on small screens, where content is below, and needs to be aligned. Circle gives us consistent alignment, numbers will differ when every step has different width:

image

File: scss/_patterns_lists.scss [hunk 1]
```diff
@@ -13,21 +14,20 @@ $list-status-icon-height: $default-icon-size;
...
+    padding-left: calc(map-get($line-heights, default-text) + $list-step-bullet-margin);
+    position: relative;
 
     &::before {
-      align-self: start;
-      background-color: $color-x-light;
-      border: 1px solid $color-x-dark;
-      border-radius: 100%;
-      content: counter(li);
+      aspect-ratio: 1 / 1;
```
Comment (ClementChaumel) [inline scss/_patterns_lists.scss:21]: This doesn't seem to do anything

**Review by ClementChaumel — COMMENTED** [review]
LGTM! I can't approve since it's originally my PR but I think it's in a good state!

File: scss/_patterns_lists.scss (hunk 1 above)
Comment (bartaz) [reply scss/_patterns_lists.scss:21]: Ah, true. I added it before I had to switch to use both width and height.

Comment (bartaz) [reply scss/_patterns_lists.scss:21]:
```suggestion
```

**Comment by lyubomir-popov** [comment]
@bartaz just one thing, can we please make the stepped list example use h2s? This way people who copy-paste from here will get a better, non-bold default. I mostly use h2s for these so chances are it will be the right choice.

**Comment by lyubomir-popov** [comment]
+1 for an easy merge after the above hasbeen adressed

**Pull request merged**